### PR TITLE
CLOUDP-98099: Add support for defaultRWconcern

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-98099

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by fmenezes — DISMISSED** [review]
LGTM, left a question, if is not relevant merge away

File: internal/convert/process_config.go [hunk 1]
```diff
@@ -257,6 +288,9 @@ func newMongosProcessConfig(p *opsmngr.Process) *ProcessConfig {
 	if p.Args26.Security != nil {
 		pc.Security = p.Args26.Security
 	}
+	if p.Args26.Security != nil {
+		pc.Security = p.Args26.Security
```
Comment (fmenezes) [inline internal/convert/process_config.go:292]: side note, Args26 is not the catchiest name is it? 😅

File: internal/convert/process_config.go [hunk 2]
```diff
@@ -232,6 +248,21 @@ func newMongosProcessConfig(p *opsmngr.Process) *ProcessConfig {
 		Name:                        p.Name,
 		SetParameter:                p.Args26.SetParameter,
 	}
+	if p.DefaultRWConcern != nil {
```
Comment (fmenezes) [inline internal/convert/process_config.go:251]: [q] is this correct? should it be the opposite?

Comment (gssbzn) [reply internal/convert/process_config.go:251]: in what sense? this converts from the ops manager struct to the cli struct, and checks if ops manager has a value then set the cli values from it

File: internal/convert/process_config.go (hunk 1 above)
Comment (gssbzn) [reply internal/convert/process_config.go:292]: no, and this looks like copy/paste, need to remove this block

**Review by fmenezes — DISMISSED** [review]

**Review by fmenezes — APPROVED** [review]
Restamp

**Pull request merged**